src/app/home/home.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="outer-wrapper">
 	<div class="wrapper">
 		<app-lazy-img src="{{headerImageUrl}}" alt="loklak Search - Distributed Social Media Message Search Engine"
-			width="125" height="145"></app-lazy-img>
+			title="loklak Search - Distributed Social Media Message Search Engine" width="125" height="145"></app-lazy-img>
```

Comment:
This appears more broader description `Distributed Social Media Search Engine`,  instead of `Distributed Social Media Message Search Engine`
what do you think?